subcommittee_policy_v3.md

```diff
@@ -314,6 +318,13 @@ Subcommittee Wind Up
   transfer their assets another entity, provided that entities
   objectives align with Linux Australia's statement of objects and
   the subcommittees objectives.
-  
+
+* Event subcommittees must supply Linux Australia the following:
+
+  * A list of attendees containing names, email addresses, gender,
```

Comment:
Just out of morbid interest, why are you interested in gender?

Reply:
The committee gets a _lot_ of pressure about increasing diversity at LA events, which is as it should be of course.  So we do things.  But we would be stumbling around in the dark if we don't measure the outcome, and since the major push about diversity is to get more women involved we really need to know if they are working or we are wasting money.

Unfortunately it's not just gender.  Diversity comes in many forms.  For example in the last month the committee has had a written request to increase diversity in there areas:

- Gender
- Orientation `[sexual orientation I presume]`
- Age
- Ethnic and linguistic background
- Ability, physical and cognitive

These are wonderfully politically correct goals, and if we are indeed under represented in these areas I'd say increasing diversity in those groups is a worthy goal and would support it.  But we don't measure it, so how can we know if we should tilt attendee financial aid toward attracting more homosexuals than people in the NDIS scheme?  I honestly don't have a clue, but I do know if we started asking for sexual orientation or proportion of income that comes from the NDIS scheme on our I cop what I suspect would be an unbearable amount of push back.  Yet without measuring it, you can't justify anything more than small measures like for example insisting venues have disability access, so you can't have a meaningful goal (by meaningful I mean you are willing to throw money at it) without measurement.

Reply:
Hi @rstuart,

The way I've seen this sort of thing asked (from the perspective of diversity) in the past, is not by asking people's gender/orientation/age/background/ability etc., but by asking if they identify as an under-represented member of a given demographic. Unfortunately I don't remember the exact wording of the questions, but for example, something like these may work:

* Do you identify as an under-represented gender? (Or "do you identify as an identity other than male?" - I like this less; it's more specific because people might not know what defines an under-represented gender, but it could be perceived as a little more inflammatory than necessary)
* Which of these age brackets do you fit into? 20-29, 30-39 etc. )some research required to find the most useful brackets to use)
* Do you identify as an under-represented sexuality? (or "not straight" - same concern with this as gender above)
* Do you identify as having any physical or cognitive disabilities?

And the *important* part, is to make these questions optional (perhaps with an explicit "prefer not to say" option) while emphasising that answering them helps with stats to improve diversity - some people won't feel comfortable outing themselves.

I can probably find some better question phrasing if you're interested; diversity and inclusion are a couple of my passions.

Also:

> These are wonderfully politically correct goals

It's not about political correctness at all. It's about equal representation, diversity, accessibility, and inclusion. Political correctness isn't an end in itself; the thing people tend to call "political correctness" is really just the act of being more aware and sensitive to the people who have put up with discrimination in the past and finally choosing to show them the same respect as everyone else.

In closing, I really appreciate the effort being put into highlighting the demographics that could be better supported in attending LA events; thanks heaps!